playbooks/group_vars/webportals.yml

```diff
@@ -178,7 +178,7 @@ webportal_allowance:
 # TODO: move to role default vars
 webportal_server_config_defaults:
   domain_name: "{{ ansible_host }}"
-  portal_modules: "{{ portal_modules | default('') }}"
+  portal_modules: ""
```

Comment:
Shouldn't we leave this out?

If we already have it in LastPass, this default will not be used.

If we do not have it, operator should define it in `hosts.ini` and we shouldn't pollute LastPass with default empty value anymore.